docs/cmd/kumactl/kumactl_generate_zone-token.md

```diff
@@ -0,0 +1,42 @@
+## kumactl generate zone-token
+
+Generate Zone Token
+
+### Synopsis
+
+Generate Zone Token that is used to prove identity of Zone egresses.
+
+```
+kumactl generate zone-token [flags]
+```
+
+### Examples
+
+```
+Generate token bound by zone
+$ kumactl generate zone-token --zone zone-1 --valid-for 24h
+$ kumactl generate zone-token --zone zone-1 --valid-for 24h --scope egress
+```
+
+### Options
+
+```
+  -h, --help                 help for zone-token
+      --scope scope...       scope of the token; can be 'egress' (default [egress])
```

Comment:
so how does it work with many scopes? `--scope ingres --scope egress` or `--scope=ingress,egress`?

Reply:
it works in both cases